### PR TITLE
feat(api): Add new cache entries

### PR DESCRIPTION
**Pull request by maciejopalinski**

### CODE REVIEW

File: src/providers/ApiProvider.js [hunk 1]
```diff
@@ -16,15 +16,15 @@ class ApiProvider {
     }
 
     comingSoon() {
-        return ApiService.request('GET', 'ComingSoon', '');
+        return ApiService.request('GET', 'ComingSoon', ' ');
```
Comment (T0ffik) [inline src/providers/ApiProvider.js:19]: I think sapces shouldn't be there, it wasn't work for me, I had sam errors

File: src/providers/ApiProvider.js [hunk 2]
```diff
@@ -16,15 +16,15 @@ class ApiProvider {
...
     }
 
     mostPopularTVs() {
-        return ApiService.request('GET', 'MostPopularTVs', '');
+        return ApiService.request('GET', 'MostPopularTVs', ' ');
```
Comment (T0ffik) [inline src/providers/ApiProvider.js:23]: Same as higher

File: src/providers/ApiProvider.js [hunk 3]
```diff
@@ -16,15 +16,15 @@ class ApiProvider {
...
     }
 
     mostPopularMovies() {
-        return ApiService.request('GET', 'MostPopularMovies', '');
+        return ApiService.request('GET', 'MostPopularMovies', ' ');
```
Comment (T0ffik) [inline src/providers/ApiProvider.js:27]: Same as higher

**Review by T0ffik — APPROVED** [review]

**Review by Mariusz-web — APPROVED** [review]

**Comment by maciejopalinski** [comment]
@T0ffik Thanks a lot for catching my bug. I just fixed it.

**Pull request merged**